.github/workflows/close-stale-issues.yaml

```diff
@@ -0,0 +1,20 @@
+name: "Close stale issues and PRs"
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+permissions:
+  # contents: write # only for delete-branch option
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6
+        with:
+          days-before-stale: 45
```

Comment:
stale implies no comments, right?
so 30 days is more than enough
and 7 days after is more than enough ... adjust?

Reply:
I'd like to keep it relatively relaxed for now. I looked at the default and what other projects do - the defaults are 60 days for stale and 7 days to close after stale. Let's go with the 45+15 for now and we can tune it more aggressively over time.